### PR TITLE
feat: cert pinning

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

implementing cert pinning for the android app
the cert and links are copied from the [iOS implementation](https://github.com/wireapp/wire-ios-mono/blob/develop/wire-ios/Configuration/Backend.bundle/production.json) 

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1213](https://10.10.124.134/job/AR-build-pipeline/1213/) **failed**.

**Comment by AndroidBob** [comment]
Build [1215](https://10.10.124.134/job/AR-build-pipeline/1215/) **failed**.

File: app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt [hunk 1]
```diff
@@ -49,14 +49,8 @@ class KaliumConfigsModule {
...
 
         return KaliumConfigs(
-            isChangeEmailEnabled = BuildConfig.ALLOW_CHANGE_OF_EMAIL,
-            isLoggingEnabled = BuildConfig.LOGGING_ENABLED,
-            blacklistHost = BuildConfig.DEFAULT_BACKEND_URL_BLACKLIST,
             fileRestrictionState = fileRestriction,
             forceConstantBitrateCalls = BuildConfig.FORCE_CONSTANT_BITRATE_CALLS,
-            developerFeaturesEnabled = BuildConfig.DEVELOPER_FEATURES_ENABLED,
-            enableBlacklist = BuildConfig.ENABLE_BLACKLIST,
-            maxAccount = BuildConfig.MAX_ACCOUNTS,
```
Comment (MohamadJaara) [inline app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt:59]: those values are not used inside kalium so 💥

**Comment by AndroidBob** [comment]
Build [1217](https://10.10.124.134/job/AR-build-pipeline/1217/) **failed**.

**Review by alexandreferris — APPROVED** [review]

File: buildSrc/src/main/kotlin/scripts/variants.gradle.kts [hunk 2]
```diff
@@ -175,7 +174,8 @@ android {
         }
 
         FeatureConfigs.values().forEach { configs ->
-            when (configs.configType) {
+            val imports = mutableListOf<String>()
+            val generatedConfig = when (configs.configType) {
```
Comment (borichellow) [inline buildSrc/src/main/kotlin/scripts/variants.gradle.kts]: why do you need these `val`s?

**Comment by AndroidBob** [comment]
Build [1249](https://10.10.124.134/job/AR-build-pipeline/1249/) **failed**.

**Comment by AndroidBob** [comment]
Build [1255](https://10.10.124.134/job/AR-build-pipeline/1255/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7390727)(7390727)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7390727)(7390727)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [1256](https://10.10.124.134/job/AR-build-pipeline/1256/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7390997)(7390997)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7390997)(7390997)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [1258](https://10.10.124.134/job/AR-build-pipeline/1258/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.4.0(7391039)(7391039)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.4.0(7391039)(7391039)-staging-compat.apk)

**Review by vitorhugods — APPROVED** [review]

**Pull request merged**